Sources/SwiftParser/Declarations.swift

```diff
@@ -1964,9 +1964,16 @@ extension Parser {
     } else {
       unexpectedAtEnd = nil
     }
+
+    var modifiers: RawModifierListSyntax? = attrs.modifiers
+    if modifiers == nil, unexpectedBeforeOperatorKeyword == nil {
```

Comment:
I check the ``unexpectedBeforeOperatorKeyword`` here to keep ``testOperatorDecl10`` produce the same diagnostics as the old parser. Not sure whether this makes sense🤔